### PR TITLE
Remove / and \ at the start of shell.setDir()

### PR DESCRIPTION
**Pull request by JakobDev**
If you change the directory with the cd command, it will remove / or \ at the start of a path. With this PR, the prompt of the shell will look the same, if you use shell.setDir().

### CODE REVIEW

File: src/main/resources/assets/computercraft/lua/rom/programs/shell.lua [hunk 1]
```diff
@@ -194,6 +194,10 @@ function shell.setDir( _sDir )
     if not fs.isDir( _sDir ) then
         error( "Not a directory", 2 )
     end
+    local sStartChar = string.sub( _sDir, 1, 1 )
```
Comment (SquidDev) [inline src/main/resources/assets/computercraft/lua/rom/programs/shell.lua:197]: Can we use `_sDir:sub(1, 1)` here too? Just so we're consistent with what's below.

**Comment by SquidDev** [comment]
Hrmr, I wonder if it'd be better just to do `fs.combine(sDir, "")` - this way we normalise file paths, and eliminate redundancies such as `rom/../`.

**Comment by Lemmmy** [comment]
Might have to make a test suite

**Comment by SquidDev** [comment]
And merged. Thank you very much!

**Pull request merged**